packages/util-package-finder/bin/index.js

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env node
+'use strict';
+
+const chalk = require('chalk');
+const program = require('commander');
+const figures = require('figures');
+const chunk = require('lodash/chunk');
+const orderBy = require('lodash/orderBy');
+
+const getPackages = require('../lib');
+
+program
+	.option('-j, --json', 'Return results as JSON')
+	.option('-s, --scope [name]', 'NPM scope', 'springernature')
+	.option('-f, --filters <items>', 'Comma seperated list of name filters', i => i.split(','))
+	.parse(process.argv);
+
+/**
+ * Sort the versions in descending order
+ * @param {Array} arr
+ * @return {Array}
+ */
+const sortVersions = arr => {
+	const list = [];
+	const sorted = orderBy(arr, item => item, ['desc']);
```

Comment:
Is this making use of something that lodash does? Or would `sorted = arr.sort((a,b) => b - a))` suffice?

Reply:
yeah I tried this, but it doesn't work with numbers of the format `1.2.3`, and I was using lodash anyway

Reply:
Sorry, I thought `arr` was in numeric form for some reason. That's really cool that lodash does that out the box

Reply:
Interestingly, to do it yourself it would be something like
`arr.sort((a, b) => b.localeCompare(a, undefined, {numeric: true, sensitivity: 'base'}))`